src/read.c

```diff
@@ -3231,6 +3231,7 @@ avifBool avifPeekCompatibleFileType(const avifROData * input)
     if (memcmp(header.type, "ftyp", 4)) {
         return AVIF_FALSE;
     }
+    AVIF_CHECK(avifROStreamHasBytesLeft(&s, header.size));
```

Comment:
this branch calls `avifROStreamReadBoxHeader` (unlike main, which calls avifROStreamReadBoxHeaderPartial`).

`avifROStreamReadBoxHeader`, already checks that there must be at least `header.size` bytes left. so we don't need this patch in this branch.

Reply:
Indeed, no need for that PR>